tools/onnx_subgraph/src/lib/partition.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "partition.h"
+#include <algorithm>
+#include <stdio.h>
+#include <stdlib.h>
+
+#define MAX_DEPTH 1000
+
+std::vector<onnx::GraphProto> Subgraphs;
```

Comment:
if you are not to accept my suggestion, please leave a comment about the reason.

this is my second time about this line.
```suggestion
std::vector<onnx::GraphProto> _subgraphs;
```

Reply:
> if you are not to accept my suggestion, please leave a comment about the reason.
> 
> this is my second time about this line.

sorry, I missed this suggestion, and I removed this line, because global define is not good, thank you